test/alphabet/aminoacid/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.2)
+project(alphabet_aminoacid_test CXX)
+
+add_executable(alphabet_aminoacid_aa27_test aa27_test.cpp)
+add_dependencies(alphabet_aminoacid_aa27_test gtest)
+target_link_libraries(alphabet_aminoacid_aa27_test ${SEQAN3_LIBRARIES})
+add_test(NAME test_alphabet_aminoacid_aa27_test COMMAND alphabet_aminoacid_aa27_test)
```

Comment:
the other submitted test did split between alphabet and alphabet_sequence so please to this here, too.